source/fundamentals/indexes.txt

```diff
@@ -144,13 +150,47 @@ created above.
    :language: js
    :start-after: begin-query
    :end-before: end-query
+   :dedent:
 
 Multikey indexes behave differently from non-multikey indexes in terms of
 query coverage, index bound computation, and sort behavior. For a full
 explanation of multikey indexes, including a discussion of their behavior
 and limitations, refer to the :manual:`Multikey Indexes page
 </core/index-multikey>` in the MongoDB manual.
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations. When you create a clustered
+index, you also create a :v5.3:`clustered collection
+</core/clustered-collections/#std-label-clustered-collections>`. 
+
+To create a clustered index, specify the ``clusteredIndex`` option in
+the ``CollectionOption`` when you create a collection. The
+``clusteredIndex`` option must specify an index on the ``_id`` field,
+and for the index to be unique.
```

Comment:
Issue: Sentence could be more concise when describing the unique index
Suggestion: "The clusteredIndex option must specify the _id field as the key and the unique field as true."